logic/src/commonMain/kotlin/com/wire/kalium/logic/sync/SyncPendingEventsUseCase.kt

```diff
@@ -0,0 +1,43 @@
+package com.wire.kalium.logic.sync
+
+import com.wire.kalium.logic.data.event.Event
+import com.wire.kalium.logic.data.event.EventRepository
+import com.wire.kalium.logic.functional.onFailure
+import com.wire.kalium.logic.functional.suspending
+import com.wire.kalium.logic.kaliumLogger
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.flow
+import kotlinx.coroutines.flow.onCompletion
+
+class SyncPendingEventsUseCase(
+    private val syncManager: SyncManager,
+    private val eventRepository: EventRepository,
+    private val conversationEventReceiver: EventReceiver<Event.Conversation>
+) {
+
+    /**
+     * @return Flow<Unit> that emits only once - when syncing Pending Events was done
```

Comment:
It doesn't actually emit anything.

It only closes when it's done, without emitting.

```kt
syncPendingEvents().collect {
    println("Collected event") // This will never happen
}

syncPendingEvents().onCompletion {
    println("Completed flow") // This will be called
}.collect()
```

So the docs are a bit misleading.

I suggest just changing it to a `suspend` function, without returning anything.